pegasus/migrations/118_add_contact_rollups_columns.rb

```diff
@@ -0,0 +1,8 @@
+Sequel.migration do
+  change do
+    alter_table(:contact_rollups) do
+      add_column :forms_submitted, String, size: 4096
+      add_column :form_roles, String, size: 4096
```

Comment:
Why is it "FORMS_submitted" and "FORM_roles", i.e., the change in pluralization?